jest.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
   testPathIgnorePatterns: [
     'cypress/.*'
   ],
+  setupFiles: [
+    './mock.js'
```

Comment:
can we not have it as a global setup, not all tests requires it, best not to keep it global. just have it within the test that needs it

Reply:
as the contributing.md explicit over implicit

Reply:
Ok it's a missing animation library when running unit tests. Will move it on the test file first